sections/leaderboard/Leaderboard/Leaderboard.tsx

```diff
@@ -307,6 +307,8 @@ const Leaderboard: FC<LeaderboardProps> = ({ compact }: LeaderboardProps) => {
 							<StyledTable
```

Comment:
This table should have the same "loading" state as above so we avoid this state:
<img width="338" alt="image" src="https://user-images.githubusercontent.com/10401554/182455804-7b32a8bf-ca08-4278-9ada-0b2866195ac7.png">